### ISSUE TITLE
minimatch always fails

### ISSUE THREAD

**Issue by eflopez** [mend/renovate-on-prem#254]
Hello! 

I can't seem to get auto discover filter to work with `whitesource/renovate:3.0.0` as the pattern is always rejected:
```
ERROR: Error initializing scheduler
       "err": {
         "message": "invalid pattern",
         "stack": "TypeError: invalid pattern\n    at assertValidPattern (/usr/src/app/node_modules/minimatch/minimatch.js:275:11)\n    at minimatch (/usr/src/app/node_modules/minimatch/minimatch.js:116:3)\n    at /usr/src/app/src/server.js:1:12975\n    at Array.filter (<anonymous>)\n    at githubScheduler (/usr/src/app/src/server.js:1:12965)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async startScheduler (/usr/src/app/src/server.js:1:9310)\n    at async /usr/src/app/src/server.js:1:7408"
       }
```

I am testing by running the docker image with an interactive shell, however its the same if its an environment variable:
`ubuntu@83a214a0ef16:/usr/src/app$ node src/server.js --autodiscover true --autodiscover-filter 'renovate-test/*'`

The resolved combined config from the logs is:

```
         "autodiscover": true,
         "autodiscoverFilter": ["renovate-test/*"]
```

Any thoughts?

**Comment by mnacken** [comment]
Maybe this has something to do with the changes in https://github.com/renovatebot/renovate/releases/tag/33.0.0

There is a listed Feature:
config: move autodiscovery filter from string to array (https://github.com/renovatebot/renovate/issues/16525) ([0a6d97f](https://github.com/renovatebot/renovate/commit/0a6d97f23a734a8bccbce78543dca2f483db0999)), closes https://github.com/renovatebot/renovate/issues/8763

**Comment by mnacken** [comment]
Maybe this has something to do with the changes in https://github.com/renovatebot/renovate/releases/tag/33.0.0

There is a listed feature:
config: move autodiscovery filter from string to array (https://github.com/renovatebot/renovate/issues/16525) ([0a6d97f](https://github.com/renovatebot/renovate/commit/0a6d97f23a734a8bccbce78543dca2f483db0999)), closes https://github.com/renovatebot/renovate/issues/8763

**Comment by rarkins** [comment]
Confirmed/reproduced. Renovate On-Prem is still expecting a string, not an array. I will publish a release with this fixed soon.

**Comment by jamietanna** [comment]
Looks like this has now been fixed? I can see a Renovate PR that's bumping to [v3.0.1 with the fix in it?](https://github.com/mend/renovate-on-prem/releases/tag/3.0.1)

**Comment by rarkins** [comment]
Always please refer to the release notes: https://github.com/mend/renovate-on-prem/releases

**Comment by jamietanna** [comment]
Thanks Rhys, I assumed that there may have been a bump of Renovate to fix the issue, looks like it's stayed the same :+1:

**Comment by morremeyer** [comment]
This is fixed with 3.0.1 indeed, if you've upgraded to 3.0.1 and it fails with `f.split is not a function`, please see #259.

**Comment by rarkins** [comment]
Fixed properly in 3.0.2
